src/db/database.py

```diff
@@ -0,0 +1,69 @@
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+from sqlalchemy.orm import sessionmaker
+from src.db.models import Base, User
+from sqlalchemy.future import select
+from sqlalchemy import update, delete
+from sqlalchemy.exc import IntegrityError, SQLAlchemyError
+
+
+# Wrapper
+class Database:
+    def __init__(self, database_url: str):
+        self.engine = create_async_engine(database_url, echo=True, future=True)
+
+        self.async_session = sessionmaker(
+            self.engine, class_=AsyncSession, expire_on_commit=False
+        )
+
+    async def init_database(self):
+        async with self.engine.begin() as conn:
+            await conn.run_sync(Base.metadata.create_all)
+
+    async def get_db(self):
+        async with self.async_session() as db:
+            return db
+
+    async def read(self, user_id: int):
```

Comment:
I think these methods need to be in a separate class which is only responsible for handling the CRUD operations with the database instead of putting them into the file which manages sessions and DB in general